packages/radix-ui-themes/src/styles/utilities/text-wrap.css

```diff
@@ -0,0 +1,19 @@
+/*
+ * We are modeling this after the "text-wrap" property, but use "white-space" because it has better support as of 2024.
+ */
```

Comment:
Everything related to text wrapping in CSS seems like such a mess historically. Apparently, `text-wrap` is supposed to be the spec now, but it's not fully supported yet:
https://caniuse.com/?search=text-wrap

`white-space` can achieve equivalent styles, but handles both whitespace collapsing and text wrapping concerns